src/lib/components/modal/CodeSnippet.tsx

```diff
@@ -233,7 +236,7 @@ execute();
                       <AceEditor
                         readOnly
                         mode={item.mode}
-                        theme="monokai"
+                        theme={CURR_THEME.jsonTheme}
```

Comment:
need to check if the specified theme are included in one of the imports, also choose a default if not included